framework/db/Schema.php

```diff
@@ -460,7 +460,7 @@ public function quoteValue($str)
             return $str;
         }
 
-        if (($value = $this->db->getSlavePdo()->quote($str)) !== false) {
+        if (mb_stripos($this->db->dsn, 'odbc:') === false && ($value = $this->db->getSlavePdo()->quote($str)) !== false) {
```

Comment:
@xpohoc69 The first check could be made more robust by calling getDriverName() https://github.com/yiisoft/yii2/blob/master/framework/db/Connection.php#L975

Reply:
@marcovtwout getDriverName() return mssql, not odbc

Reply:
Ah, so we can only rely on the DSN to check if we are using ODBC?

Reply:
Yes, I think so